modules/red-hat-marketplace-features.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * applications/red-hat-marketplace.adoc
+
+[id="red-hat-marketplace-features_{context}"]
+= Red Hat Marketplace features
+
+Cluster administrators can use link:https://marketplace.redhat.com/en-us/documentation/getting-started[Red Hat Marketplace] to manage software on {product-title}, give developers self-service access to deploy application instances, and correlate application usage against a quota.
+
+[id="marketplace-clusters_{context}"]
+== Connect {product-title} clusters to the Marketplace
+
+You can install a common set of applications on {product-title} clusters that you connect to the Marketplace. You can also use the Marketplace to track cluster usage against your subscriptions or quota. Users that you add by using the Marketplace have their product usage tracked and billed to your organization.
```

Comment:
This is the only section that starts with "you". Everywhere else says cluster admins or developers. Just wanted to check if it needed to be only admins or anything.

Reply:
@robszumski I'd guess that this is admins only. Am I right?

Reply:
yes, this is admins